<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Configuration Parameters</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>alpha</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Drop-in Rate</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Drop-out Rate</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Race</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="raceComboBox">
       <item>
        <property name="text">
         <string>ALL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AFRICAN_AMERICAN</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CAUCASIAN</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HISPANIC</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="alphaDoubleSpinBox">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="dropInRateDoubleSpinBox">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="dropOutRateDoubleSpinBox">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>IDB Probabilities</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>0 Alleles</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>1 Allele</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>2 Alleles</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="zeroAlleleProbDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="oneAlleleProbDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="twoAlleleProbDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Probabilities to Calculate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="OS0U">
       <property name="text">
        <string>One Suspect, No Unknowns</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="OS1U">
       <property name="text">
        <string>One Suspect, One Unknown</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="OS2U">
       <property name="text">
        <string>One Suspect, Two Unknowns</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="NS1U">
       <property name="text">
        <string>No Suspect, One Unknown</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="NS2U">
       <property name="text">
        <string>No Suspect, Two Unknowns</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="NS3U">
       <property name="text">
        <string>No Suspect, Three Unknowns</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>641</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>881</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Load From File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputFileLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="inputBrowseButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>881</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>File to Output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputFileLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="outputBrowseButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="outputButton">
       <property name="text">
        <string>(Save and) Go!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
